packages/js/src/TelnyxRTC.ts

```diff
@@ -126,4 +130,58 @@ export default class TelnyxRTC extends TelnyxRTCClient {
   newCall(options: ICallOptions) {
     return super.newCall(options);
   }
+
+  /**
+   * Checks if the running browser has support for TelnyRTC
+   *
+   * @return an object with telnyx webrtc information.
+   *
+   * @examples
+   *
+   * If your browser has supported
+   *
+   * ```js
+   * const info = TelnyxRTC.webRTCInfo();
+   * console.log(info) // => 'This browser does not support @telnyx/webrtc. \nPlease, use Chrome|Firefox|Safari.'
+   * ```
+   *
+   * ### Error handling
+   *
+   * An error will be thrown if your browser doesn't supported TelnyxRTC
+   *
+   * ```js
+   * const info = TelnyxRTC.webRTCInfo();
+   * console.log(info) // => {browserName: "Chrome", browserVersion: 87, supportWebRTC: true, supportWebRTCAudio: true, …}
+   * ```
+   */
+  public static webRTCInfo() {
+    return getWebRTCInfo();
+  }
+
+  /**
+   * Returns the WebRTC supported browser list.
+   *
+   * The following table indicates the browsers supported by TelnyxRTC.
+   * We support the most recent (N) and the two previous (N-2) versions of these browsers unless otherwise indicated.
+   *
+   * |  | Chrome | Firefox | Safari | Edge |
+   * |---|---|---|---|---|
+   * | Android | [x] | [x] | [ ] | [ ] |
+   * | iOS | [x] | [x] | [x] | [ ] |
+   * | Linux | [x] | [x] | [ ] | [ ] |
+   * | MacOS | [x] | [x] | [x] | [x] |
+   * | Windows | [x] | [x] | [ ] | [x] |
+   *
+   * @return an array with supported operational systems and browsers.
+   *
+   * @examples
+   * 
+   * ```js
+   * const browserList = TelnyxRTC.webRTCSupportedBrowserList();
+   * console.log(browserList) // => [{"operationSystem": "Android", "supported": [{"browserName": "Chrome", "features": ["video", "audio"], "supported": "full"},{...}]
+   * ```
+   */
+  public static webRTCSupportedBrowserList() {
```

Comment:
Would be nice to have an interface here too:
```suggestion
  public static webRTCSupportedBrowserList(): Object[] {
```

Reply:
I will do that